retrofit/src/main/java/retrofit/RequestBuilder.java

```diff
@@ -366,6 +367,9 @@ Request build() {
         body = formEncodingBuilder.build();
       } else if (multipartBuilder != null) {
         body = multipartBuilder.build();
+      } else if (Utils.methodRequiresRequestBody(requestMethod)) {
+        // Body is absent, make an empty body.
+        body = RequestBody.create(MediaType.parse("text/plain"), NO_BODY);
```

Comment:
MIME type is a question. Converter know about it. But it's not capable of producing an empty body. If we delefate this to a Converter, it will have to provide something like `RequestBody emptyBody()`.
On the other hand, if API accepts an empty body, content type should really matter.